_tuning-your-cluster/availability-and-recovery/snapshots/snapshot-restore.md

```diff
@@ -296,6 +296,62 @@ Use the following steps to register a snapshot repository backed by an Azure sto
    }
    ```
 
+### Set up Microsoft Azure Blob Storage
+
+To use Microsoft Azure Blob Storage as a snapshot repository, install the `repository-azure` plugin on all nodes using the following command:
+
+```bash
+./bin/opensearch-plugin install repository-azure
+```
+
+To use `repository-azure` plugin after installation, define your Azure Blob Storage settings before initializing the node.
+
+Define your Azure Storage account name using the following secure setting:
+
+```bash
+./bin/opensearch-keystore add azure.client.default.account
+```
+
+Then, use one of the following options to set up your account keys for Azure Blob Storage.
+
+#### Azure storage account key
+
+Use the following setting to specify your Azure Storage account key:
+
+```bash 
+./bin/opensearch-keystore add azure.client.default.key
+```
+
+#### Shared access signatures tokens
+
+Use the following setting when accessing Azure with a shared access signature (SAS) token:
+      
+```bash
+./bin/opensearch-keystore add azure.client.default.sas_token      
+```
+
+#### Azure credential token 
+
+If you're using a non-SAS token credential or an account key, use the following steps:
+
+1. Define your Azure storage account name in secure setting:
+
+   ```bash
+   ./bin/opensearch-keystore add azure.client.default.account
+   ```
+
+2. Add your token credential type to `opensearch.yml`, using either the `managed` or `managed_identity` values:
```

Comment:
What do these options do? Would only `managed_identity` be supported based on the limitation described below?

Reply:
@Naarcha-AWS, thank you for review and fixing it. I have added a small paragraph to explain managed identity supported in opensearch. I hope that would be clearer.